scanner/fetcher.go

```diff
@@ -108,16 +113,23 @@ func (f *Fetcher) Prepare(ctx context.Context) (*ct.SignedTreeHead, error) {
 	return sth, nil
 }
 
-// Run performs fetching of the Log. Blocks until scanning is complete or
-// context is cancelled. For each successfully fetched batch, runs the fn
-// callback.
+// Run performs fetching of the Log. Blocks until scanning is complete, the
+// passed in context is canceled, or Stop is called. For each successfully
```

Comment:
Could you expand the comment to make clear that calling Stop() doesn't instantaneously terminate the blocking call.  (Maybe just "Blocks until scanning is complete, the passed in context is canceled, or (after finishing pending work) Stop is called." ?)

Reply:
Done. I've slightly reordered your suggestion, as calling the `Stop` precedes finishing pending work.